### PR TITLE
Spotless code formatter

### PR DESCRIPTION
**Pull request by yschimke**
Apply spotless formatter and confirm within Intellij

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,6 +57,7 @@ plugins {
   id 'ru.vyarus.animalsniffer' version '1.5.0'
   id 'com.github.johnrengelman.shadow' version '4.0.1'
   id 'me.champeau.gradle.japicmp' version '0.2.6'
+  id "com.diffplug.gradle.spotless" version "3.13.0"
```
Comment (yschimke) [inline build.gradle]: remove?

File: build.gradle [hunk 2]
```diff
@@ -104,6 +108,14 @@ subprojects { project ->
     signature 'net.sf.androidscents.signature:android-api-level-21:5.0.1_r2@signature'
     signature 'org.codehaus.mojo.signature:java18:1.0@signature'
   }
+  
+  spotless {
+    kotlin {
+      target "**/*.kt"
+      ktlint(versions.ktlint).userData(['indent_size': '2', 'continuation_indent_size' : '2'])
+//      licenseHeaderFile project.rootProject.file('spotless/copyright.kt')
```
Comment (yschimke) [inline build.gradle]: How to apply conditionally?

**Comment by JakeWharton** [comment]
Looks good so far

**Comment by yschimke** [comment]
`./gradlew spotlessApply`

File: okhttp/src/main/java/okhttp3/internal/platform/Jdk8WithJettyBootPlatform.kt [hunk 3]
```diff
@@ -24,11 +24,16 @@ import javax.net.ssl.SSLSocket
 
 /** OpenJDK 8 with `org.mortbay.jetty.alpn:alpn-boot` in the boot class path.  */
 class Jdk8WithJettyBootPlatform(
-  private val putMethod: Method, private val getMethod: Method, private val removeMethod: Method,
-  private val clientProviderClass: Class<*>, private val serverProviderClass: Class<*>
+  private val putMethod: Method,
+  private val getMethod: Method,
+  private val removeMethod: Method,
+  private val clientProviderClass: Class<*>,
+  private val serverProviderClass: Class<*>
```
Comment (swankjesse) [inline okhttp/src/main/java/okhttp3/internal/platform/Jdk8WithJettyBootPlatform.kt]: yes, much nicer here

**Pull request merged**